src/pages/dmt5/_dhero.js

```diff
@@ -0,0 +1,359 @@
+        }
+    }
+`
+
+const DHero = ({
+    title,
+    background_alt,
+    background_svg,
+    background_svg2,
+    content,
+    image_name,
+    is_mobile,
+    join_us_for_free,
+    go_to_live_demo,
+    Logo,
+}) => {
+    const data = useStaticQuery(query)
+    const handleRedirect = () => {
+        const path = image_name === 'dbot' ? '/bot' : '/'
+        window.open(deriv_app_url + path, '_blank')
+    }
+    const DLogo = styled(Logo)`
+        margin-right: 1.6rem;
+    `
+    const BackgroundSVG = styled(background_svg)`
+        position: absolute;
+        top: 170px;
+        right: 0;
+
+        @media ${device.laptopM} {
+            width: 48%;
+            max-width: 492px;
+            height: initial;
+        }
+        @media ${device.laptop} {
+            width: 50%;
+        }
+        @media ${device.tabletL} {
+            width: 45%;
+            max-width: 350px;
+        }
+        @media ${device.tablet} {
+            width: 70%;
+        }
+        @media ${device.tabletS} {
+            width: 80%;
+            max-width: 337px;
+        }
+        @media ${device.mobileL} {
+            max-width: 250px;
+            min-height: 244px;
+        }
+        @media ${device.mobileM} {
+            max-width: 205px;
+            min-height: 0;
+        }
+    `
+    const BackgroundSVG2 = styled(background_svg2)`
+        position: absolute;
+        top: 0;
+        right: 214px;
+
+        @media ${device.laptopM} {
+            width: 48%;
+            max-width: 492px;
+            height: initial;
+        }
+        @media ${device.laptop} {
+            width: 50%;
+        }
+        @media ${device.tabletL} {
+            width: 45%;
+            max-width: 350px;
+        }
+        @media ${device.tablet} {
+            width: 70%;
+        }
+        @media ${device.tabletS} {
+            width: 80%;
+            max-width: 337px;
+        }
+        @media ${device.mobileL} {
+            max-width: 250px;
+            min-height: 244px;
+        }
+        @media ${device.mobileM} {
+            max-width: 205px;
+            min-height: 0;
+        }
+    `
+    return (
+        <Wrapper>
+            <BackgroundSVG />
+            <BackgroundSVG2 />
+            <InformationWrapper height="unset" direction="column">
+                <StyledHeader as="h1" weight={500}>
+                    <DLogo />
+                    {title}
+                </StyledHeader>
+                <HeroContent>
+                    <StyledContent as="h2">{content}</StyledContent>
+                </HeroContent>
+                <LinkWrapper>
+                    {join_us_for_free && (
+                        <DemoButton secondary="true" to="/signup/">
+                            {localize('Create free demo account')}
+                        </DemoButton>
+                    )}
+                    {go_to_live_demo && (
+                        <GoToLiveDemo tertiary onClick={handleRedirect}>
```

Comment:
based on this [comment](https://github.com/binary-com/deriv-com/pull/980/files#r465471017):
<LocalizedLink external tertiary to={handleRedirect()}>